### PR TITLE
Patient portal

### PR DESCRIPTION
**Pull request by teryhill**
Patient Portal. First pass . #418

### CODE REVIEW

**Comment by tmccormi** [comment]
First pass indicates some testing is requested... right?

**Comment by teryhill** [comment]
Yes I need you guys to test this. I am working on some more changes also.

File: library/js/xl/jquery-datetimepicker-2-5-4-alternate.js.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?php
```
Comment (aethelwulffe) [inline library/js/xl/jquery-datetimepicker-2-5-4-alternate.js.php]: Is there any issue with the libraries and the assets directory at this time?

File: patients/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# My project's README
```
Comment (aethelwulffe) [inline patients/README.md]: Will any of this file need to be updated in the Docs elsewhere?

File: library/js/xl/jquery-datetimepicker-2-5-4-alternate.js.php (hunk 1 above)
Comment (teryhill) [inline library/js/xl/jquery-datetimepicker-2-5-4-alternate.js.php]: No . that is OpenEMR's date picker. I will convert it over to ours when I get some time

File: patients/README.md (hunk 2 above)
Comment (teryhill) [inline patients/README.md]: It will need to be written

**Comment by aethelwulffe** [comment]
Terry, we already have conflicts on this.  Please resolve them so we can get this moved in pretty quick.  I know that several other things, including additional stuff with the portal, cleanup of legacy stuff, date-picker, JQUERY and a million other things are waiting on this to get merged.  I wish it were a modules directory thing, but we need to get the slick newness of a very standard core feature (which this is) so that we have a more error-free patient portal.  I have not had luck with the existing code myself.  It had been noted that documents show up in the wrong patients and all kinds of other stuff, making the old code essentially unusable to careful people.

**Comment by teryhill** [comment]
@aethelwulffe  Conflicts resolved. This can wait to be merged if you want , It is kinda self contained.

**Comment by aethelwulffe** [comment]
I still would rather it moved forward as expediently as possible.  We just need a third set of eyes on it to merge.

**Comment by tmccormi** [comment]
Third Eye... signing in ...

**Comment by tmccormi** [comment]
Quick question.  Does this replace the other 3 patient portals?

**Comment by teryhill** [comment]
since we don't have ZH's or CMS 8^) 

This portal has a lot of functionality and I think anything that may be missing to get rid of the others can be added. There is some work still being done for the CCDA documents.

**Comment by tmccormi** [comment]
so, without going thru 306 files have the older portals been removed from globals then?

**Comment by teryhill** [comment]
Yes they have and from the code. We will need to hammer on this and make sure we do not have any issues.

**Pull request merged**